src/core/list/components/Title.jsx

```diff
@@ -0,0 +1,40 @@
+import React from 'react'
+import { Box, Typography } from '@material-ui/core'
+import { makeStyles } from '@material-ui/core/styles'
+
+const primary = {
+  fontSize: 36,
+  fontWeight: '700',
+  lineHeight: 1.11,
+  letterSpacing: '-0.03em',
+  textAlign: 'left',
+  marginBottom: 10,
+}
+const secondary = {
+  fontSize: 16,
+  fontWeight: '400',
+  lineHeight: 1.625,
+  letterSpacing: '0em',
```

Comment:
After a zero length, the unit identifier is optional
https://www.w3.org/TR/CSS22/syndata.html#length-units

Reply:
Done.